CMakeLists.txt

```diff
@@ -1830,7 +1830,12 @@ MYSQL_CHECK_LDAP()
 MYSQL_CHECK_LDAP_DLLS()
 
 # Percona LDAP Simple / LDAP SASL authentication plugins
-OPTION(WITH_PERCONA_AUTHENTICATION_LDAP "Build with Percona LDAP Simple / LDAP SASL authentication plugins" ON)
+# Problem with  MacOS system libraries and complains for compilation error ldap_set_urllist_proc
+IF(APPLE_ARM)
```

Comment:
I think this affects all APPLE builds, not just ARM.

Reply:
removed the ARM condition